examples/plugins/react-native-idfa-plugin/ios/IdfaPlugin.swift

```diff
@@ -0,0 +1,42 @@
+import AdSupport
+import AppTrackingTransparency
+import Foundation
+
+@objc(IdfaPlugin)
+class IdfaPlugin: NSObject {
```

Comment:
Thanks for implementing this @falconandy.

I think I would prefer to use a 3rd party RN NPM module for the IDFA access.

That way don't need to maintain/test this functionality ourselves and it will be easier for users to add our example to their project (just copy 1 file and add the npm dependency). wdyt?

cc @yuhao900914 @liuyang1520

Reply:
@justin-fiedler Yes, we can adopt the example plugin to use a 3rd party package.
Pros: easier to setup for users.
Cons: new dependency.

Also we could provide both options (with/without 3rd party package).

Reply:
Ready